<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>docscreenClass</class>
 <widget class="QMainWindow" name="docscreenClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1050</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>docscreen</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #000;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="liveView" name="liveGraphicsView">
    <property name="geometry">
     <rect>
      <x>316</x>
      <y>16</y>
      <width>1920</width>
      <height>1080</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="infoGroupBox">
    <property name="geometry">
     <rect>
      <x>2478</x>
      <y>2</y>
      <width>81</width>
      <height>1438</height>
     </rect>
    </property>
    <widget class="QLabel" name="logoLabel">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>4</y>
       <width>70</width>
       <height>282</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QFrame" name="statusLabel">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>300</y>
       <width>71</width>
       <height>835</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DINPro-Medium</family>
       <pointsize>36</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="timeFieldLabel">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>1174</y>
       <width>71</width>
       <height>259</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DINPro-Medium</family>
       <pointsize>36</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
   <widget class="captureListView" name="previewListview">
    <property name="geometry">
     <rect>
      <x>91</x>
      <y>984</y>
      <width>2377</width>
      <height>429</height>
     </rect>
    </property>
    <property name="flow">
     <enum>QListView::LeftToRight</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="footerGroupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>81</width>
      <height>1438</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <widget class="QFrame" name="deviceLabel">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>730</y>
       <width>71</width>
       <height>703</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DINPro-Medium</family>
       <pointsize>36</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="catheterViewLabel">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>6</y>
       <width>71</width>
       <height>445</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DINPro-Medium</family>
       <pointsize>36</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="zoomFactorLabel">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>548</y>
       <width>71</width>
       <height>331</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DINPro-Medium</family>
       <pointsize>36</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="recordingLabel">
    <property name="geometry">
     <rect>
      <x>2402</x>
      <y>1148</y>
      <width>71</width>
      <height>287</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DINPro-Medium</family>
      <pointsize>36</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <zorder>infoGroupBox</zorder>
   <zorder>footerGroupBox</zorder>
   <zorder>liveGraphicsView</zorder>
   <zorder>previewListview</zorder>
   <zorder>recordingLabel</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>captureListView</class>
   <extends>QListView</extends>
   <header>Widgets/capturelistview.h</header>
  </customwidget>
  <customwidget>
   <class>liveView</class>
   <extends>QGraphicsView</extends>
   <header>liveview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>2</number>
  </property>
  <property name="gridDeltaY">
   <number>2</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
